website/routes.py

```diff
@@ -1227,6 +1228,13 @@ def make_url_map(app):
         ),
 
         Rule(
+            '/settings/subscribe/',
```

Comment:
For deleting subscriptions, I'm sending a signal when a node is deleted (https://github.com/samanehsan/osf.io/blob/feature/notifications/website/project/model.py#1438). How would I change this to use a `DELETE ` endpoint?